pypeit/core/extract.py

```diff
@@ -971,8 +972,38 @@ def fit_profile(image, ivar, waveimg, thismask, spat_img, trace_in, wave, flux,
     return (profile_model, xnew, fwhmfit, med_sn2)
 
 
+def parse_manual(manual_par):
+    """
+    Parse the rather klunky ManualExtractionPar parameters into more useful items
+
+    Args:
+        manual_par (:class:`pypeit.par.pypeitpar.ManualExtractionPar`):
+            Manual extract parameter object
+
+    Returns:
+        tuple: spats, specs, det, fwhm
+
+    """
+    if isinstance(manual_par['det'], list):
+        spat_spec = manual_par['spat_spec'].split(',')
+        det = [int(obj) for obj in manual_par['det'].split(',')]
+        fwhm = [float(obj) for obj in manual_par['fwhm'].split(',')]
```

Comment:
I tested manual extraction. 
manual_par['spat_spec'], manual_par['det'] and manual_par['fwhm'] are already lists, so .spit(',') does not work here. After I removed it, the extraction worked well.

Reply:
edited